my_phase-retrieval/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.11)
+project(cuPhaseRet LANGUAGES CXX CUDA)
+
+find_package(PythonInterp 3.6 REQUIRED)
+find_package(PythonLibs 3.6 REQUIRED)
+list(APPEND CMAKE_PREFIX_PATH "D:/Anaconda3/Lib/site-packages/pybind11/share/cmake/pybind11")
+find_package(pybind11 REQUIRED)
+
+
+list(APPEND CMAKE_PREFIX_PATH "D:/opencv/build")
```

Comment:
The `CMAKE_PREFIX_PATH` should not be set inside the `CMakeLists.txt`. This is a environment variable of your system.

Reply:
Please check all `CMakeLists.txt` in the folder. I found it a least 4 times.